FSW/src/headers/hkUtil.hpp

```diff
@@ -0,0 +1,52 @@
+#ifndef HKUTIL_H
+#define HKUTIL_H 
+
+/* - - - - - - Includes - - - - - - */
+// C++ libraries
+
+// Other libraries
+
+// NS2 headers
+#include "config.hpp"
+
+
+/* - - - - - - Declarations - - - - - - */
+void handleHousekeeping();
+void setHeater();
+void sampleHousekeepingData();
+float voltageToTemp(float voltage);
+void timeSortHkData();
+
+struct HousekeepingData // stores one point of housekeeping data
```

Comment:
Make sure structs like these are documented well outside of the code for reference when future teams (or us later in testing) are looking at the data.

Reply:
I will update the readme with info on all the data structures, and more generally to provide an overview of how the software fits together for people who are not me and Jaret.

Reply:
See new issue